mon-pix/app/controllers/user-tutorials-v2/saved.js

```diff
@@ -7,4 +7,8 @@ export default class SavedController extends Controller {
     { label: '50', value: 50 },
     { label: '100', value: 100 },
   ];
+
+  unloadRecord(tutorial) {
+    return tutorial.unloadRecord();
```

Comment:
**question:** Est-ce qu'on est certain que c'est la bonne façon de faire ?

En lisant la doc de [`unloadRecord`](https://api.emberjs.com/ember-data/3.28/classes/Model/methods/unloadRecord?anchor=unloadRecord) j'ai l'impression qu'on s'appuie sur un effet de bord et ça me semble un peu dangereux :

> Unloads the record from the store. This will not send a delete request to your server, it just unloads the record from memory.

Ce qu'on veut vraiment c'est retirer le record du model de la route `saved` et non le retirer du store.